plaidml/edsl/edsl.h

```diff
@@ -987,11 +988,17 @@ class gather {
     return *this;
   }
 
+  gather& paddingMode(PaddingMode mode) {
```

Comment:
If it is padding mode for all interpolated mode, we have to add this mode for nearest, linear and cubic interpolated gather not only linear interpolation. 
An alternative is to use LinearInterpolationPaddingMode,  NearestInterpolationPaddingMode,  CubicInterpolationPaddingMode. Not sure if we have a better solution.

Reply:
I think LinearInterpolationPaddingMode is better, because i don't know whether it is necessary to add PaddingMode for nearest and cubic interpolated gather.